### PR TITLE
Website: Testnet Page

### PR DESCRIPTION
**Pull request by MartinMinkov**
This PR implements the Testnet page. Please note that not all the button links are correctly hooked up. 

image
image
image
image

image
image
image

### CODE REVIEW

File: frontend/website-redesign/src/pages/Testnet.re [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+[@react.component]
+let make = () => {
+  <Page title="Mina Protocol" footerColor=Theme.Colors.orange>
+    <div className=Styles.page>
+      <div className=Nav.Styles.spacer />
+      <Hero
+        title="Testnet"
+        header="Secure the Network"
+        copy={j|"Push the boundaries of Mina’s testnet to help prepare for mainnet."|j}
```
Comment (michellewong793) [inline frontend/website-redesign/src/pages/Testnet.re:108]: delete the quotation marks here, they're showing up in the header and dont need to be!

File: frontend/website-redesign/src/pages/Testnet.re [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            <Icon kind=Icon.ArrowRightSmall />
+          </Button>
+        </div>
+        <div className=Styles.gradientSection>
+          <a href="/leaderboard" className=Styles.leaderboardLink>
+            <Leaderboard interactive=false />
+          </a>
+        </div>
+        <p className=Styles.disclaimer>
+          {React.string(
```
Comment (michellewong793) [inline frontend/website-redesign/src/pages/Testnet.re:136]: the spacing between the table and the disclaimer only needs to be 48px 
image

File: frontend/website-redesign/src/pages/Testnet.re [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+             "Testnet Points are designed solely to track contributions to the Testnet and are non-transferable. Testnet Points have no cash or monetary value and are not redeemable for any cryptocurrency or digital assets. We may amend or eliminate Testnet Points at any time.",
+           )}
+        </p>
+      </Wrapped>
+      <FeaturedSingleRow
```
Comment (michellewong793) [inline frontend/website-redesign/src/pages/Testnet.re:141]: two comments: is that the right image size? and can you change the font style below Testnet Challenges to be letterSpacing(-0.4px) and the monument grotesk mono font? 

image

Comment (michellewong793) [reply frontend/website-redesign/src/pages/Testnet.re:141]: Screen Shot 2020-09-24 at 3 43 13 PM
Design for reference^ the "Learn how..."

File: frontend/website-redesign/src/pages/Testnet.re [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+          button: {
+            FeaturedSingleRow.Row.buttonText: "See the Latest Challenges",
+            buttonColor: Theme.Colors.mint,
+            buttonTextColor: Theme.Colors.digitalBlack,
+            dark: true,
+          },
+        }
+      />
+      <TestnetRetroModule />
+      <ButtonBar
```
Comment (michellewong793) [inline frontend/website-redesign/src/pages/Testnet.re:160]: can you change the ButtonBar paddingY to 5.75rems?

**Review by figitaki — APPROVED** [review]

**Pull request merged**